OrbitQt/orbitmainwindow.cpp

```diff
@@ -242,6 +244,44 @@ void OrbitMainWindow::SetupCodeView() {
   OrbitCodeEditor::setFileMappingWidget(ui->FileMappingWidget);
 }
 
+//-----------------------------------------------------------------------------
+void OrbitMainWindow::ShowFeedbackDialog() {
+  QDialog feedback_dialog{nullptr, Qt::WindowTitleHint | Qt::WindowCloseButtonHint};
+
+  const auto layout = QPointer{new QGridLayout{&feedback_dialog}};
+  const auto button_box =
+      QPointer{new QDialogButtonBox{QDialogButtonBox::StandardButton::Close}};
```

Comment:
What do you think about using `Cancel` instead of `Close`?

Reply:
I don't really have a preference. It seems you think it's a better choice (right?), could you elaborate why?

Reply:
The&mdash;well&mdash;cancel button in dialogs usually has the "Cancel" text, that's what I'm personally used to.

Reply:
Hey, as discussed over chat, I'll leave it as it is right now, as there's not a big difference for users here. It's clear what "close" will do in this case, as would be "cancel".